### DISCUSSION TITLE
Silk Touch Reinforced Deepslate

### DISCUSSION THREAD

**Discussion by RhythmicSys** [PurpurMC/Purpur#1245] [__Completed__] [answered]
Reinforced deepslate has a breaking time of 82.5 seconds, and does not drop
This block would be really cool in some builds and it's properties would make it super neat for some farms
I think it'd be cool for it to drop when mined with a silk touch tool

**Comment by LeStegii** [comment]
I like this idea, as there already is a feature like this for spawners, I don't see why it wouldn't fit Purpur :)

**Comment by granny** [comment, marked as answer]
Implemented as a data pack, https://modrinth.com/datapack/purpurpacks-silk-touch-reinforced-deepslate
